### DISCUSSION TITLE
Tailwindcss colors not working in adonisjs?

### DISCUSSION THREAD

**Discussion by MichaelBelgium** [adonisjs/core#4277] [Help] [answered]
I followed [this ](https://tailwindcss.com/docs/guides/adonisjs) page, but not all colors are working somehow? Eg text-white is working but not text-green-800, bg-green-500 etc

Am i missing something?

EDIT: i am using htmx that loads a view after a http request like:

```
@set('color', user === null ? 'red' : 'green')

<div class="p-4 text-sm text-{{ color }}-800 rounded-lg bg-{{ color }}-50 dark:bg-{{ color }}-800 dark:text-{{ color }}-400" role="alert">
```

Is it that why it can't detect the colors i used?

**Comment by RomainLanz** [comment, marked as answer]
Hey @MichaelBelgium! 👋🏻 

The issue is the way you construct your classes.
https://tailwindcss.com/docs/content-configuration#dynamic-class-names

**Reply by MichaelBelgium** [reply]
Aah that explains it, thanks!
